install man pages without gzipping them
don't install spring-legacy.6, this is obsolete per upstream and there
is no spring-legacy binary

Index: doc/CMakeLists.txt
--- doc/CMakeLists.txt.orig
+++ doc/CMakeLists.txt
@@ -43,10 +43,10 @@ if    (UNIX)
 		set(man_src
 				"${MAN_SOURCE_DIR}/spring.6.txt")
 		set(man_dst
-				"${MAN_BUILD_DIR}/spring-dedicated.6.gz"
-				"${MAN_BUILD_DIR}/spring-headless.6.gz"
-				"${MAN_BUILD_DIR}/spring-legacy.6.gz"
-				"${MAN_BUILD_DIR}/spring.6.gz")
+				"${MAN_BUILD_DIR}/spring-dedicated.6"
+				"${MAN_BUILD_DIR}/spring-headless.6"
+				#"${MAN_BUILD_DIR}/spring-legacy.6"
+				"${MAN_BUILD_DIR}/spring.6")
 		set_source_files_properties(${man_dst} PROPERTIES GENERATED TRUE)
 
 		add_custom_command(
